react/features/room-lock/components/RoomLockPrompt.web.js

```diff
@@ -0,0 +1,18 @@
+import { Component } from 'react';
```

Comment:
Please drop this since @damencho is working on dialogs on React. Also, why is this needed for toolbars?